components/Dialogs/src/MDCAlertController.m

```diff
@@ -495,6 +495,12 @@ + (NSString *)bundlePathWithName:(NSString *)bundleName {
 - (void)viewWillLayoutSubviews {
   [super viewWillLayoutSubviews];
 
+  if (self.titleColor == nil) {
+    self.alertView.titleColor = UIColor.blackColor;
+  }
+  if (self.messageColor == nil) {
+    self.alertView.messageColor = UIColor.blackColor;
+  }
```

Comment:
Can this logic be moved to the titleColor and messageColor setters and in `setupAlertView`?

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/8588/commits/aeb7921cc2e2ef88d89a6e09763753f841daf9a0